ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh

```diff
@@ -150,6 +150,14 @@ else
   KUBECONFIG_SRC=/root/${TYPE}/kubeconfig
 fi
 
+collect_ai_logs() {
+  echo "Collecting AI logs ..."
+  AI_CLUSTER_ID=$(curl -sS http://$bastion2:8080/api/assisted-install/v2/clusters/  | jq -r .[0].id)
+  curl -Lo "${ARTIFACT_DIR}/ai-cluster-logs-$AI_CLUSTER_ID.tar"  http://$bastion2:8080/api/assisted-install/v2/clusters/$AI_CLUSTER_ID/logs
+}
+
+trap 'collect_ai_logs' EXIT
```

Comment:
Have you checked  if those logs have any kind of sensitive information?

Reply:
Not more than a must-gather :see_no_evil: